client/src/components/pages/profileBusiness/index.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react'
+import { Form, Input, AutoComplete, Cascader, Select } from 'antd'
+
+import Button from 'components/utils/Button'
+import { residences, BusinesTypeValues } from './static'
+import Options from "components/utils/option"
+import json from './country.json'
```

Comment:
@AbdallahAmmar96 It might help to have a more descriptive import name like `import countryJson from './country.json'`